### DISCUSSION TITLE
How can I login?

### DISCUSSION THREAD

**Discussion by jason-liu22** [mirumee/saleor-dashboard#2718] [Q&A] [answered]
I setup this `saleor-dashboard` project on my local machine.
And I have a saleor instance running.
I setup my account with google(signin with google) on the saleor cloud.
But the login page to dashboard has no option to use my account.
So I tried to invite my another work email to the project in the saleor clound.
I received a invitation successfully.
But when I click the `Accept Invite` button from the email, it redirect me to `https://cloud.saleor.io`.
So I have no option to add an staff properly, and eventually I can't login to dashboard.
What can I do to login dashboard?

**Comment by karolkielecki** [comment, marked as answer]
Hi @endo-aki22, 
To login to your dashboard:

1. Once the project/sandbox has been created, you need to set up a password for your dashboard -- open the environment/sandbox details page. 
2. Click the reset password button, and you should receive an email with instructions on how to set a password. (See the screenshot below).
3. Log in to your dashboard.

Please let me know if you still can't log in to your dashboard. 

![image](https://user-images.githubusercontent.com/57907951/205068506-3c27ee59-5dfe-47a7-86da-e968c62efa20.png)
